front/package.json

```diff
@@ -11,11 +11,14 @@
     "@testing-library/jest-dom": "^4.2.4",
     "@testing-library/react": "^10.4.8",
     "@testing-library/user-event": "^7.2.1",
+    "apexcharts": "^3.22.3",
     "axios": "^0.19.2",
+    "eslint": "^7.15.0",
     "http-proxy-middleware": "^1.0.6",
```

Comment:
eslint should not be in package.json since it comes included with create-react-app:

1. Remove eslint from package.json
2. delete node_modules and package-lock.json
3. run npm install